website/content/docs/connect/cluster-peering/usage/create-cluster-peering.mdx

```diff
@@ -0,0 +1,834 @@
+---
+layout: docs
+page_title: Establish Cluster Peering Connections
+description: >-
+  Generate a peering token to establish communication, export services, and authorize requests for cluster peering connections. Learn how to create, list, read, check, and delete peering connections.
+---
+
+# Establish cluster peering connections
```

Comment:
the file name is "create" but the title is "establish". should we make these more consistent?